### ISSUE TITLE
on mac, better to install using easy_install instead of pip

### ISSUE THREAD

**Issue by itaybiancovonage** [KeepSafe/android-resource-remover#33]
easy_install seems to behave nicer with el-capitan than pip. i suggest to change the ReadMe accordingly.

**Comment by manyata** [comment]
this is true. the "android-resource-remover" command is not recognised until you use easy_install

**Comment by ca77y** [comment]
should also help with #36 if there is anything we can do better

**Comment by ca77y** [comment]
encoding problems as well #38
